### ISSUE TITLE
Video does not play in Firefox

### ISSUE THREAD

**Issue by kraebsli** [h5p/h5p-interactive-book#136]
I have an issue with interactive videos inside the interactive book. In firefox the interactive video does not play in non full screen mode. The only error get in firefox console: "No ajax path found"

any ideas?

Kathrin

**Comment by kraebsli** [comment]
Update: The video does not play because the interactive book contains math jax code.

**Comment by otacke** [comment]
I assume this is related to your issue: https://h5p.org/comment/50297#comment-50297

**Comment by kraebsli** [comment]
that's right, we solved the problem now and updated to Math Display 1.0.7.
